frontend/src/metabase-types/api/mocks/presets/sample_database.ts

```diff
@@ -64,6 +64,28 @@ export const REVIEWS = {
   CREATED_AT: 71,
 };
 
+// Note: don't assign field values to the field object itself
+// Field values are not included in the field object in the API response
+// Please use `setupFieldValuesEndpoints` utility from `__support__/server-mocks`
+
+export const PRODUCT_CATEGORY_VALUES: FieldValues = {
```

Comment:
I would use the same pattern with `createProductCategoryFieldValues` and not expose constants directly, but not sure